### PR TITLE
feat: update package exports and use bare imports everywhere

### PR DESCRIPTION
**Pull request by Knorgias**
## What I did

1.

### CODE REVIEW

File: packages/checkbox-group/define.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { LionCheckboxGroup } from './src/LionCheckboxGroup.js';
+import { LionCheckboxIndeterminate } from './src/LionCheckboxIndeterminate.js';
+import { LionCheckbox } from './src/LionCheckbox.js';
+
+customElements.define('lion-checkbox-group', LionCheckboxGroup);
+customElements.define('lion-checkbox-indeterminate', LionCheckboxIndeterminate);
+customElements.define('lion-checkbox', LionCheckbox);
```
Comment (jorenbroekema) [inline packages/checkbox-group/define.js:7]:
```suggestion
import '@lion/checkbox-group/define-checkbox-group';
import '@lion/checkbox-group/define-checkbox';
import '@lion/checkbox-group/define-checkbox-indeterminate';
```

so then end user can do

```js
import '@lion/checkbox-group/define';

// or

import '@lion/checkbox-group/define-checkbox-group';
```

File: packages/form-integrations/test/form-reset.test.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import '@lion/button/lion-button';
-import '@lion/checkbox-group/lion-checkbox';
-import '@lion/checkbox-group/lion-checkbox-group';
+import '@lion/button/define';
+import '@lion/checkbox-group/define-checkbox';
+import '@lion/checkbox-group/define-checkbox-group';
```
Comment (jorenbroekema) [inline packages/form-integrations/test/form-reset.test.js:3]:
```suggestion
import '@lion/checkbox-group/define';
```

File: packages/form-integrations/test/helpers/umbrella-form.js [hunk 3]
```diff
@@ -8,8 +8,8 @@ import '@lion/input-datepicker/lion-input-datepicker';
 import '@lion/input-amount/lion-input-amount';
 import '@lion/input-iban/lion-input-iban';
 import '@lion/input-email/lion-input-email';
-import '@lion/checkbox-group/lion-checkbox-group';
-import '@lion/checkbox-group/lion-checkbox';
+import '@lion/checkbox-group/define-checkbox-group';
+import '@lion/checkbox-group/define-checkbox';
```
Comment (jorenbroekema) [inline packages/form-integrations/test/helpers/umbrella-form.js:12]:
```suggestion
import '@lion/checkbox-group/define';
```

File: packages/form-integrations/test/model-value-consistency.test.js [hunk 4]
```diff
@@ -12,8 +12,8 @@ import '@lion/input-iban/lion-input-iban';
 import '@lion/input-range/lion-input-range';
 import '@lion/textarea/lion-textarea';
 
-import '@lion/checkbox-group/lion-checkbox-group';
-import '@lion/checkbox-group/lion-checkbox';
+import '@lion/checkbox-group/define-checkbox-group';
+import '@lion/checkbox-group/define-checkbox';
```
Comment (jorenbroekema) [inline packages/form-integrations/test/model-value-consistency.test.js:16]:
```suggestion
import '@lion/checkbox-group/define';
```

**Review by jorenbroekema — CHANGES_REQUESTED** [review]

File: packages/calendar/test-helpers/index.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export * from './CalendarObject.js';
```
Comment (daKmoR) [inline packages/calendar/test-helpers/index.js:1]: please define every export (e.g. no star exports) for better support of bundlers

File: packages/combobox/docs/listboxData.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
+export const listboxData = [
```
Comment (jorenbroekema) [inline packages/combobox/docs/listboxData.js:1]: Copy from packages/listbox. We were importing it from listbox before but we don't have an export maps entry for it so WDS could not resolve it. Imo, just copying is fine for this...

File: .storybook/main.js [hunk 7]
```diff
@@ -1,44 +1,25 @@
...
+      './{packages,packages-node}/*/docs/*.md',
+      './{packages,packages-node}/*/docs/!(assets)**/*.md',
+      './{packages,packages-node}/*/docs/validate/!(assets)**/*.md',
+      './packages/helpers/*/README.md',
+      './docs/README.md',
+      './docs/**/*.md',
+      './README.md',
+      './demo/README.md',
+      './demo/docs/*.md',
+    ])
```
Comment (jorenbroekema) [inline .storybook/main.js:18]: With globby like this we can at least use JS to filter out stuff that the glob caught that we don't want, makes it a bit easier than puzzling for multiple days on a glob pattern that works 😓

**Review by jorenbroekema — APPROVED** [review]

**Comment by jorenbroekema** [comment]
If we can accept that our storybook code snippets are gone I suppose we could merge, but not having copy-able things in our demos is a shame 😓

**Comment by daKmoR** [comment]
has been incorporated into the `chore/staticDocs` branch

**Pull request closed without merging**